test/jdk/java/lang/ProcessHandle/PermissionTest.java

```diff
@@ -207,6 +208,11 @@ public void setBasicPermissions() {
         permissions.add(new RuntimePermission("createSecurityManager"));
         permissions.add(new PropertyPermission("testng.show.stack.frames",
                 "read"));
+        permissions.add(new PropertyPermission("testng.thread.affinity", "read"));
+        permissions.add(new PropertyPermission("testng.mode.dryrun", "read"));
+        permissions.add(new PropertyPermission("testng.report.xml.name", "read"));
+        permissions.add(new ReflectPermission("suppressAccessChecks"));
+        permissions.add(new PropertyPermission("testng.timezone", "read"));
```

Comment:
might be better to group the testng.* properties so they aren't mixed up with the other runtime permissions.